tests/loop_tool/actions_test.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""Integrations tests for the loop_tool CompilerGym environment."""
+
+import compiler_gym
+from tests.test_main import main
+
+
+def test_basic():
+    env = compiler_gym.make("looptool-v0")
```

Comment:
Could you refactor this (and other `test_xxx()` funcs) to use the pattern:

```py
with compiler_gym.make("looptool-v0") as env:
    # test body
```

Using the `with` syntax ensures that `env.close()` is called so that no compiler gym service is orphaned.